torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -1177,6 +1177,49 @@ Operation listAdd(const Node* node) {
     return 0;
   };
 }
+template <class TList, class TElement>
+Operation listMulIntLeft(const Node* node) {
```

Comment:
It should be possible to write:
```
int64_t listMultIntLeft(Stack& stack) {
...
}
```

The Node* argument is only needed for ops that actually use it.